sysl2/sysl/seqs/fmtParser_test.go

```diff
@@ -128,7 +128,7 @@ func TestParseWithEqualConditionValue(t *testing.T) {
 	formatStr := fp.Parse(attrs)
 
 	// Then
-	assert.Equal(t, "1ba%%(DONOTWANT?//bc//\n|bb)****", formatStr)
+	assert.Equal(t, "1ba%%(DONOTWANT?//bc//\\n|bb)****", formatStr)
```

Comment:
Regexes generally work better with back-quote strings.

Reply:
Changed to back-quote.